.github/workflows/ci.yml

```
@@ -318,6 +318,34 @@ jobs:
             -DTBB_CPF=${{ matrix.preview }} -DTBB_TEST=OFF -DTBB_EXAMPLES=ON ..
           cmake --build . -v --target light_test_examples
 
+  linux-examples-testing-docker:
+    name: examples_${{ matrix.container_name }}_${{ matrix.cxx_compiler }}_cxx${{ matrix.std }}_${{ matrix.build_type }}_preview=${{ matrix.preview }}
+    runs-on: ['ubuntu-latest']
```

```suggestion
    runs-on: ['uxl-gpu-xlarge']
```
might as well run it on a GPU if available. Note this change should cause the issue https://github.com/uxlfoundation/oneTBB/issues/1760 for the cpu examples.